companies/joyn-gmbh.json

```
@@ -0,0 +1,21 @@
+{
+    "slug": "joyn-gmbh",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "entertainment"
+    ],
+    "name": "Joyn GmbH",
+    "address": "Ridlerstrasse 57\n80339 Munich\nGermany",
+    "phone": "+49 800 4395696",
+    "fax": "+49 89 3977199",
```

```suggestion
    "fax": "+49 89 943977199,
```
that's the number from the "impressum"